hypha/apply/dashboard/static/js/select2.multi-checkboxes.js

```diff
@@ -3,77 +3,124 @@
  * - allow to select multi values via normal dropdown control
  *
  * author      : wasikuss
- * repo        : https://github.com/wasikuss/select2-multi-checkboxes
+ * repo        : https://github.com/wasikuss/select2-multi-checkboxes/tree/amd
  * inspired by : https://github.com/select2/select2/issues/411
  * License     : MIT
  */
-(function($) {
-  var S2MultiCheckboxes = function(options, element) {
-    var self = this;
-    self.options = options;
-    self.$element = $(element);
-  var values = self.$element.val();
-    self.$element.removeAttr('multiple');
-    self.select2 = self.$element.select2({
-      allowClear: true,
-      minimumResultsForSearch: options.minimumResultsForSearch,
-      placeholder: options.placeholder,
-      closeOnSelect: false,
```

Comment:
The select popup get closed as soon as you make a selection, this did not happen before and this setting might be what fixed that.

Reply:
This was kind of overwriting closeOnSelect options but in the latest version, it is amd based build and everything is moduler, so select-multi-checkboxes just gives you utils not options you have to pass yourself.